apps/src/templates/manageStudents/ManageStudentsAgeCell.jsx

```diff
@@ -0,0 +1,43 @@
+import React, {Component, PropTypes} from 'react';
+import {ages} from '../AgeDropdown';
+
+class ManageStudentAgeCell extends Component {
+  static propTypes = {
+    id: PropTypes.number.isRequired,
+    age: PropTypes.number,
+    isEditing: PropTypes.bool,
+  };
+
+  state = {
+    ageValue: this.props.age
+  };
+
+  onChangeAge = (e) => {
+    this.setState({ageValue: e.target.value});
+  }
+
+  render() {
+    const {age} = this.props;
+    return (
+      <div>
+        {!this.props.isEditing &&
+          <div>
+            {age}
+          </div>
+        }
+        {this.props.isEditing &&
+          <select
+            ref={element => this.root = element}
```

Comment:
Does this ref get used at all (suspect this was a consequence of trying to be like AgeDropdown?)